src/lib/conda.ml

```diff
@@ -0,0 +1,123 @@
+(*
+  Conda is a Python environment and package manager:
+  http://conda.pydata.org/docs/
+
+  We use it to ensure a consistent Python environment for tools that depend
+  on Python.
+*)
+
+open Common
+open Run_environment
+
+let rm_path = Workflow_utilities.Remove.path_on_host
+
+let dir ~meta_playground = meta_playground // "conda_dir"
+let commands ~meta_playground com = dir ~meta_playground // "bin" // com
+let bin = commands "conda"
+let activate = commands "activate"
+let deactivate = commands "deactivate"
+
+(* give a conda command. *)
+let com ~meta_playground fmt =
+  Printf.sprintf ("%s " ^^ fmt) (bin ~meta_playground)
+
+(* A workflow to ensure that conda is installed. *)
+let installed ~host ~meta_playground =
+  let open KEDSL in
+  let url =
+    "https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh"
```

Comment:
Not really, only that 3 is more "forward" looking. The environment that is defined/configured below actually uses `Python2.7`, with the idea being that eventually we'll be able parameterize by stable environment.